### PR TITLE
Improved Label Error Detection with xval comparison

### PR DESCRIPTION
**Pull request by ulya-tkch**
Benchmarking label error detection for cross validation vs single pass training.

### CODE REVIEW

**Comment by ulya-tkch** [comment]
Issue: for food datasets, train_dataset.classes contains an extra class called images that should be removed with `train_dataset.classes.remove("images")`. Removal should be tested.
